src/kubelet/metric/transform.go

```diff
@@ -0,0 +1,54 @@
+package metric
+
+import (
+	"fmt"
+
+	"github.com/newrelic/nri-kubernetes/v2/src/definition"
+)
+
+// OneMetricPerLabel transforms a map of labels to FetchedValues type,
+// which will be converted later to one metric per label.
+// It also prefix the labels with 'label.'
+func OneMetricPerLabel(rawLabels definition.FetchedValue) (definition.FetchedValue, error) {
+	return PrefixFromMapString("label.")(rawLabels)
```

Comment:
Now that `PrefixFromMapString` is not used directly anywhere, what's the point of refactoring it?

Also, it's a good practice to move code in one commit, then change it in another, so the changes are nicely highlighted. Right now,   `MapToPrefix` -> `PrefixFromMapString` rename is hidden.

Reply:
Agreed, I missed out that opportunity. I was a bit annoyed by the whole `interface{}` thing tricking me I kind of rushed the commits.